web/html/javascript/spacewalk-checkall.js

```diff
@@ -187,20 +187,26 @@ function process_single_checkbox(cboxes, checkall) {
 }
 
 function update_server_set(variable, set_label, checked, values) {
-    DWRItemSelector.select(set_label, values, checked, makeAjaxHandler(pageResponse));
+  const data = {
+    label: set_label,
+    values,
+    checked
+  }
+  ajax('item-selector', data, pageResponse, "application/json")
 }
 
 function pageResponse(data) {
-    var resp = eval(data);
-    if (typeof(resp['header']) != 'undefined') {
-            dwr.util.setValue("header_selcount", resp.header, {escapeHtml: false});
-    }
-    if (document.getElementById('pagination_selcount_top')) {
-      dwr.util.setValue("pagination_selcount_top", resp.pagination);
-      if (jQuery('#pagination_selcount_bottom').lenght > 0) {
-        dwr.util.setValue("pagination_selcount_bottom", resp.pagination);
-      }
+  var resp = JSON.parse(data)
+  resp = JSON.parse(resp)
+  if (typeof(resp['header']) != 'undefined') {
+    jQuery("#header_selcount").html(resp.header);
+  }
+  if (document.getElementById('pagination_selcount_top')) {
+    jQuery("#pagination_selcount_top").html(resp.pagination);
+    if (jQuery('#pagination_selcount_bottom').lenght > 0) {
```

Comment:
This is a typo, and a good example of why this code would be better ported to Typescript instead: `lenght` → `length`.